test/browser.js

```diff
@@ -3,6 +3,6 @@
 
 // require('./exchange-files')
 // require('./pubsub')
-// require('./kad-dht')
```

Comment:
not a target of this PR, @dryajov

Reply:
Those tests are being skipped either way - I added it to browser tests since it actually loads fine in the browser. I added it for consistency, but can remove if needed.